packages/oas-extensions/src/index.ts

```diff
@@ -0,0 +1,134 @@
+import type Oas from 'oas';
+import type Operation from 'oas/operation';
```

Comment:
Over in `oas-extensions` this was originally this:

```ts
// eslint-disable-next-line import/no-named-default
import type { default as Oas, Operation } from 'oas';
```

But because `Operation` is no longer being exported there[^1] I had to change this.

[^1]: https://github.com/readmeio/oas/pull/799/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L901